src/retry/mempool.rs

```diff
@@ -0,0 +1,16 @@
+//! Transaction validation logic.
```

Comment:
let's add a mention to Reth, saying we are copy pasting because we can't import in this specific case